src/app/static/src/app/components/plots/MapAdjustScale.vue

```diff
@@ -17,22 +17,26 @@ import {ColourScaleType} from "../../store/colourScales/colourScales";
         </div>
 
         <div class="mt-2 ml-2">
-            <div class="row p-0 mb-2">
-                <label for="custom-min-input" class="col col-form-label col-2">Min</label>
-                <div class="col pt-1">
-                    <input id="custom-min-input" type="number" step="any" v-model.number="colourScaleToAdjust.customMin"
-                           @change="update" @keyup="update" :disabled="disableCustom">
+            <form novalidate>
```

Comment:
Need this to stop the inputs complaining with warning message when you manually enter a value into a numeric input which isn't 'legal' according to the step value e.g. 0.5 if step is 1